apps/backoffice-v2/src/common/components/atoms/Input/Input.tsx

```diff
@@ -11,6 +11,9 @@ export const Input = forwardRef<HTMLInputElement, TInputProps>(
         className={ctw(
           'flex h-10 w-full rounded-md border border-input bg-transparent px-3 py-2 text-sm ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50',
           className,
+          {
+            'h-4 w-4': type === 'checkbox',
```

Comment:
I think we should handle checkboxes like we handle select, links, and so on. Render ShadCN's checkbox component if `type === 'checkbox'`.